### ISSUE TITLE
[COGNITION]: STEM Automated Denial - Content should be simplified for usability

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#22585]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor, Peggy_

## User Story or Problem Statement

While testing the content with NVDA, I noticed the screen reader continued to read nested lists when the virtual cursor was on a parent list item. This created very long paragraphs of text being read aloud. It was difficult to understand the relationship between main ideas and nested ones.

## Details

I'm cross-linking to the content feedback https://github.com/department-of-veterans-affairs/va.gov-team/issues/22441#issuecomment-811278061 because that guidance will improve this item as well.

## Acceptance Criteria

- [ ] Feedback in #22441 has been implemented or team comments about decision not to implement

**Comment by af-manish** [comment]
Spike - @hallihocine to test if this is happening on VoiceOver on Mac.
